exchanges/order/order_types.go

```diff
@@ -170,7 +170,7 @@ const (
 	PartiallyCancelled Status = "PARTIALLY_CANCELLED"
 	PartiallyFilled    Status = "PARTIALLY_FILLED"
 	Filled             Status = "FILLED"
-	Cancelled          Status = "CANCELED"
+	Cancelled          Status = "CANCELLED"
```

Comment:
The tests will need to be updated based on this change: https://github.com/thrasher-corp/gocryptotrader/blob/master/exchanges/order/order_test.go#L504